Exercises/exercise_1_btc_transfer_cli.md

```diff
@@ -153,7 +153,7 @@ To send wrapped Bitcoin back to Bitcoin, run the following commands:
   ```
   You can then copy the raw transaction and send it to bitcoin testnet with bitcoin's JSON-RPC API, or a web interface such as https://live.blockcypher.com/btc/pushtx/
 
-🛑 **IMPORTANT: Confirm the outpoint that returns the change (when in doubt, confirm all outpoints of this transaction)**
+🛑 **IMPORTANT: Confirm the outpoint that returns the change (when in doubt, try to confirm each outpoint of this transaction. Any outpoint that is not the change outpoint will immediately abort the confirmation.)**
```

Comment:
i'm not sure users will under what the "change output is". 
- I would still ask the users to confirm all outputs.
"In this step, you will verify all UTXO outputs. Most of the verifications should fail, and you will see `failed to execute message; message index: 0: outpoint address unknown`" on the verifications for individual withdrawal outputs. One of the verifications should succeed (this is the output with the remaining balance that returned back to the axelar network key)."